src/constants/colours.ts

```diff
@@ -22,15 +22,20 @@ export const BORDER_COLOURS = {
     purple: 'border-purple border-2',
 } as const satisfies Record<Colour, string>;
 
+// the list below is sorted in alphabetical order by tech stack name
 export const TECH_COLORS = {
     'Discord.py': '#7387CE',
     FastAPI: '#F6BF00',
+    NextUI: '#A8DADC',
     'Next.js': '#B17CA6',
+    'Pragmatic Drag & Drop': '#FF6F61',
     Python: '#64B550',
     React: '#ED8C9B',
+    Requests: '#FFD166',
```

Comment:
This colour is to similar to the colour used for FastAPI, can you please change the colour. The dark blue colour you used previously would be fine.